zone/guild_mgr.cpp

```diff
@@ -334,40 +349,40 @@ void ZoneGuildManager::DescribeGuild(Client *c, uint32 guild_id) const {
-		auto can_speak_guild_chat = info->ranks[guild_rank].permissions[GUILD_SPEAK] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
-		auto can_invite = info->ranks[guild_rank].permissions[GUILD_INVITE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
-		auto can_remove = info->ranks[guild_rank].permissions[GUILD_REMOVE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
-		auto can_promote = info->ranks[guild_rank].permissions[GUILD_PROMOTE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
-		auto can_demote = info->ranks[guild_rank].permissions[GUILD_DEMOTE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
-		auto can_set_motd = info->ranks[guild_rank].permissions[GUILD_MOTD] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
-		auto can_war_peace = info->ranks[guild_rank].permissions[GUILD_WARPEACE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
-		popup_text += fmt::format(
-			"<tr>"
-			"<td>{} ({})</td>"
-			"<td>{}</td>"
-			"<td>{}</td>"
-			"<td>{}</td>"
-			"<td>{}</td>"
-			"<td>{}</td>"
-			"<td>{}</td>"
-			"<td>{}</td>"
-			"<td>{}</td>"
-			"</tr>",
-			!info->ranks[guild_rank].name.empty() ? info->ranks[guild_rank].name : "Nameless",
-			guild_rank,
-			can_demote,
-			can_hear_guild_chat,
-			can_invite,
-			can_promote,
-			can_remove,
-			can_set_motd,
-			can_speak_guild_chat,
-			can_war_peace
-		);
-	}
+					
+	//for (uint8 guild_rank = 0; guild_rank <= GUILD_MAX_RANK; guild_rank++) {
+	//	auto can_hear_guild_chat = info->rank_s[guild_rank].permissions[GUILD_HEAR] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	auto can_speak_guild_chat = info->ranks[guild_rank].permissions[GUILD_SPEAK] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	auto can_invite = info->ranks[guild_rank].permissions[GUILD_INVITE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	auto can_remove = info->ranks[guild_rank].permissions[GUILD_REMOVE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	auto can_promote = info->ranks[guild_rank].permissions[GUILD_PROMOTE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	auto can_demote = info->ranks[guild_rank].permissions[GUILD_DEMOTE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	auto can_set_motd = info->ranks[guild_rank].permissions[GUILD_MOTD] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	auto can_war_peace = info->ranks[guild_rank].permissions[GUILD_WARPEACE] ? "<c \"#00FF00\">Y</c>" : "<c \"#F62217\">N</c>";
+	//	popup_text += fmt::format(
+	//		"<tr>"
+	//		"<td>{} ({})</td>"
+	//		"<td>{}</td>"
+	//		"<td>{}</td>"
+	//		"<td>{}</td>"
+	//		"<td>{}</td>"
+	//		"<td>{}</td>"
+	//		"<td>{}</td>"
+	//		"<td>{}</td>"
+	//		"<td>{}</td>"
+	//		"</tr>",
+	//		!info->ranks[guild_rank].name.empty() ? info->ranks[guild_rank].name : "Nameless",
+	//		guild_rank,
+	//		can_demote,
+	//		can_hear_guild_chat,
+	//		can_invite,
+	//		can_promote,
+	//		can_remove,
+	//		can_set_motd,
+	//		can_speak_guild_chat,
+	//		can_war_peace
+	//	);
+	//}
```

Comment:
?

Reply:
The old guild system had a popup to display info that appeared to mirror the RoF permission system.  I should remove this as the permission system displays the data now.